bciers/package.json

```diff
@@ -74,21 +74,21 @@
     "@babel/preset-typescript": "^7.22.11",
     "@mui/material-nextjs": "^5.15.11",
     "@nx-tools/nx-container": "5.2.0",
```

Comment:
We can update the `nx-container` plugin that we use, it's had a [version bump to v6](https://github.com/gperdomor/nx-tools/releases/tag/nx-container%406.0.1) to support nx 19 as well

Reply:
Thanks for that, I bumped that to match the rest of our nx setup 🙇